src/markdown/privacy/privacy.md

```diff
@@ -562,7 +585,7 @@ If you wish to raise a complaint on how we have handled your personal data, you
 
 #### Right to lodge a complaint with a relevant supervisory authority {#right-to-lodge-a-complaint-with-a-relevant-supervisory-authority}
 
-We encourage you to contact us at privacy@cc0de.dev if you have any privacy related concerns. Should you disapprove of the response we have provided you, you have the right to lodge a complaint with our supervisory authority, or with the data protection authority of the European member state you live or work in. The details of the supervisory authority responsible for Berlin, Germany, are:
```

Comment:
This seems to have been added by accident at some point. `cc0x.dev` is the correct one.